docs/contributing/coding-style.md

```diff
@@ -297,6 +297,51 @@ All errors should define in [internal/errors package](https://github.com/vdaas/v
 
 Please use [internal/errgroup](https://github.com/vdaas/vald/blob/master/internal/errgroup) for synchronized error handling on multi-goroutine processing.
 
+### Error checking
+
+All functions return `error` if the function can fail. It is very important to ensure the error checking is performed.
+To reduce human mistake that missing the error checking, please check the error using the following style:
+
+```golang
+// good
+if err := fn(); err != nil {
```

Comment:
this code is correct when there is only one error handling.
but when there is two error handling, I think the following is correct.

```go
err := fn()
if err != nil {

}

err = fn2()
if err != nil {

}

```
If you do not reuse the error, the variable will be declared again.

Reply:
how about this? but I think it is ok to create another `err` variable in `fn2()`, what do you think?

```suggestion
err := fn()
if err != nil {
    // handle err
}

if err = fn2(); err != nil {
    // handle err
}
```

Reply:
@kevindiu 
in my opinion, this rule is not necessary.
And, we should not design many coding rules for each section.
It too hard to remember all of the rules for me 😓

Reply:
@vankichi so do you think the original guideline is ok? 
```go
// good
if err := fn(); err != nil {
    // handle error
}
```

Reply:
it's ok

Reply:
> this rule is not necessary.

I think so too.
The more rules we have, the harder it is to learn and the more complicated it becomes.